service/ui_plugin_build_baron.go

```diff
@@ -271,13 +225,6 @@ func (uis *UIServer) makeNotification(project string, t *task.Task) (*notificati
 	return n, nil
 }
 
-type searchReturnInfo struct {
-	Issues      []thirdparty.JiraTicket `json:"issues"`
-	Search      string                  `json:"search"`
-	Source      string                  `json:"source"`
-	FeaturesURL string                  `json:"features_url"`
-}
-
 type suggester interface {
```

Comment:
I think you can delete this and most of what's below as well right? Or would that be too messy? I'd be nice to not have this duplicated. Since DAG doesn't use the suggester anymore we will probably end up deleting this anyway